docs/remote-runners.md

```diff
@@ -26,6 +26,14 @@ When using remote runners, even though the build executes remotely, the followin
 
 To get started with free remote runners managed by Earthly, check out [Earthly Satellites](cloud/satellites.md).
 
-To get started with self-managed remote runners, see
-* The [remote buildkit page](ci-integration/remote-buildkit.md)
-* The [Kubernetes setup page](ci-integration/guides/kubernetes.md)
+To get started with self-hosted runners, see the [Self-Hosted Satellites Guide](cloud/satellites/self-hosted.md).
+
+If your use case cannot tolerate any cloud integration, then self-hosting a remote Buildkit is the best approach. Remote Buildkit has less features, is less secure, and is more difficult to deploy than Self-Hosted Satellites (see diagram below for comparison). To get started self-hosting Buildkit, see the [remote buildkit page](ci-integration/remote-buildkit.md).
+
+Below is a comparison of the different features available with each kind of remote runner.
+
+| Type | Runner Location | Cloud Control-Plane | TLS Encryption By Default | Auto-Sleep | Auto-Updates |
+| --- | --- | --- | --- | --- | --- |
+| Cloud Satellites | Earthly Cloud | ✅ Yes | ✅ Yes | ✅ Yes | ✅ Yes |
+| Self-Hosted Satellites | Your Infrastructure | ✅ Yes | ❌ No | ❌ No | ❌ No |
+| Remote Buildkit | Your Infrastructure | ❌ No Control Plane | ❌ Requires manual configuration | ❌ No | ❌ No |
```

Comment:
@thtmnisamnstr @vladaionescu what do you think of the comparison diagram? Any columns we should add/remove?

Reply:
I was also picturing how `Earthly Dedicated` could be slotted in here with checkmarks in the auto-sleep and auto-update columns.

Reply:
I think this is good. We can add more columns with more features in the future if we want.

Reply:
<img width="691" alt="Screen Shot 2024-02-12 at 15 18 29" src="https://github.com/earthly/earthly/assets/3247216/b9d46729-c46c-4a52-b3bb-ac03c6e5ad47">
The text wrapping doesn't look great

Reply:
A few thoughts...

1. We should flip rows and columns. Then we can add as many features as we want without wrapping issues. It's more common anyway to have the products as columns in such tables (and it would maintain consistency with our other comparison table).

2. I would rename the `TLS` feature to `Managed TLS Certificates`, perhaps. Then we can say just "No" for Remote Buildkit. I think a stronger message (i.e. a clear "No") here is important, because it's pretty difficult to do this yourself, and it's not immediately obvious why.

3. I wonder if we should add the following additional differences:
  * `Managed By`. It would be "You" for Self-hosted and remote buildkit and Earthly for Cloud. For Dedicated, in the future it will also be "Earthly".
  * `Cache persistence`. For Cloud - it's marked as "✅ Yes", for the others, I would put "🟡 Needs configuration". The reason to show this is that sometimes people don't realize that these are not trivial to manage because they tend to be more like pets rather than cattle.

4. Possible option: Add the Earthly Dedicated column now - sales cycles are anyway pretty long for those.

Reply:
Good suggestions, thanks. I reworked the chart but did not add dedicated, as it seemed confusing without having the corresponding docs to explain what it is.